lib/repositories/messages_repository.dart

```diff
@@ -89,6 +103,9 @@ class MessagesRepository {
       queryParameters['page_token'] = afterMessageId;
       queryParameters['direction'] = 'future';
     }
+    if(withExistedFiles == true) {
+      queryParameters['filter'] = 'files';
+    }
     if (threadId == null) {
```

Comment:
What about files in the threads, do we show them?

Reply:
Currently the API hasn’t  supported yet, let's wait for them

Reply:
I'll ping Romaric about this